modules/sdk-core/src/bitgo/tss/ecdsa/types.ts

```diff
@@ -16,9 +17,12 @@ export type SignatureShare = ECDSA.SignRT;
 export type Signature = ECDSA.Signature;
 
 export enum SendShareType {
+  AShare = 'AShare',
```

Comment:
This interface is actually for shares which we are sending to bitgo (ie: K, Mu and SS Shares), may be we can rename it to make it more apparent

Reply:
it makes sense to keep this type only send to bitgo types. reverting this change.